probert/storage.py

```diff
@@ -68,7 +68,7 @@ def model(self):
 
     @property
     def serial(self):
-        return self._get_hwvalues(['ID_SERIAL', 'ID_SERIAL_SHORT'])
+        return self._get_hwvalues(['ID_SERIAL', 'ID_SERIAL_SHORT', 'DM_UUID'])
```

Comment:
DM_UUID doesn't imply multipath, rather a disk managed by device-mapper.  lvm, raid , crypt may all have a DM_UUID.

I don't think we should equate DM_UUID as a "disk" serial. I think we should have dm property that indicate what sort of disk it is (lvm, raid), and I think multipath can be detected via multipath tools (or implicitly via duplicate serial numbers).

Reply:
True.

It's just in subiquity UI, `/dev/dm-0` is not descriptive at all, as to what it is. And I want to display a WWN somehow, which for multipath devices is available in multiple ways, one of which is DM_UUID. However subiquity UI can get to, and display DM_UUID without requiring it to be in the serial field. Currently all other DM devices are filtered out on the subiquity side of probe(), but that may change in the future.

Reply:
FWIW, curtin can use mpath-$ID_SERIAL just fine:

>>> from curtin import block
>>> import glob, os
>>> glob.glob('/dev/disk/by-id/*mpath-36005076306ffd6b60000000000002406*')
>>> os.path.realpath(glob.glob('/dev/disk/by-id/*mpath-36005076306ffd6b60000000000002406*')[0])
'/dev/dm-2'
>>> block.lookup_disk('mpath-36005076306ffd6b60000000000002406')
'/dev/dm-2'

Reply:
ack.